app/src/main/java/com/epam/themes/collectionviews/recyclerview/StudentEditFragment.java

```diff
@@ -0,0 +1,72 @@
+package com.epam.themes.collectionviews.recyclerview;
+
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.DialogFragment;
+import android.support.v7.app.AlertDialog;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.EditText;
+import com.epam.themes.collectionviews.StudentsActivity;
+import com.epam.cleancodetest.R;
+import com.epam.themes.backend.entities.Student;
+
+public class StudentEditFragment extends DialogFragment {
+    private static final String STUDENT_ARG = "student_argument";
+
+    private Student updatedStudent;
+
+    public static StudentEditFragment newInstance(Student student) {
+        StudentEditFragment dialog = new StudentEditFragment();
+        Bundle args = new Bundle();
+        StudentParcelable studentParcelable =
+            new StudentParcelable(student.getId(), student.getName(), student.getEmail(), student.getAddress());
```

Comment:
It's not related to current project, but looks like you should have only one Student model where you implement Parcelable interface. StudentParcelable can be removed